src/acl.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import os
+import random
+import json
+import string
+import re
+from jinja2 import Template
+
+class ACL:
+    def __init__(self, mode):
+        self._config = json.loads(open('/opt/sherpa/config.json', 'r').read())
```

Comment:
It is common practice and safer to use `with` https://docs.python.org/3/reference/compound_stmts.html#the-with-statement as the python interpreter is not always going to close your file. It will cleanly handle this and you can open multiple files like the one like on line 13 and handle errors gracefully if the file fails to open or DNE.